session/portforward.go

```diff
@@ -234,15 +247,16 @@ func portForwardHandlerNew(message *ws.ProtoMsg, w ResponseWriter) error {
 		ConnectionID:   connectionID,
 		ResponseWriter: w,
 		mutexAck:       &sync.Mutex{},
+		portForwarders: h.portForwarders,
 	}
 
 	key := message.Header.SessionID + "/" + connectionID
-	portForwarders[key] = portForwarder
+	h.portForwarders[key] = portForwarder
```

Comment:
`SessionID` will be the same for all entries in this map now, so we might as well index by the connectionID.
```suggestion
	h.portForwarders[connectionID] = portForwarder
```

Reply:
Right! https://github.com/mendersoftware/mender-connect/pull/15/commits/b666f62245939059bcf0c89369adea52fa3665fa